Cargo.toml

```diff
@@ -17,7 +17,8 @@ members = [
 resolver = "2"
 
 [workspace.dependencies]
-anyhow = { version = "1" }
+anyhow = { version = "^1" }
+async-stream = { version = "~0.3" }
```

Comment:
I learned late that there are implicit semver ranges in Cargo that more or less mean what we intended here. So, I think we can remove `^` and `~`.

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio